src/terra-dev-site/doc/gettingStarted.a.doc.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started
+
+Terra-dev-site offers a quick site to host test examples and documentation for your react components or md documentation.
+
+## Quick start:
+
+### Installation:
+``` bash
+npm install terra-dev-site
+```
+
+### Pre Reqs:
+* terra-toolkit 3.0.0 or greater
+* react 16
+* react-dom 16
+* README.md in the root project directory
+* Package.json in the root project directory
+* es6 code in src
+* transpiled code in lib
+
+### Running terra-dev-site
+Terra-dev-site automatically gets built when you use the provided webpack config or continently, the tt:serve command provided by terra-toolkit. If you have a custom webpack config, you can either extend the provided webpack config.
+
+Add the following command to your package.json.
```

Comment:
Add the following **terra-toolkit** command to your package.json. **See the [tt:serve docs](https://github.com/cerner/terra-toolkit/tree/master/scripts/serve#cli) for more information on the command options.**